src/Sulu/Bundle/WebsiteBundle/EventSubscriber/AnalyticsSerializeEventSubscriber.php

```diff
@@ -50,5 +50,11 @@ public function onPostSerialize(ObjectEvent $event)
             $value->domains = true;
             $event->getVisitor()->visitProperty($metadata, $value, $event->getContext());
         }
+
+        $content = $analytics->getContent();
```

Comment:
i have realized that this will happen when you have installed dependencies from https://github.com/wachterjohannes/sulu/pull/10